code/game/machinery/computer/HolodeckControl.dm

```diff
@@ -453,6 +455,14 @@
 	return 0
 
 /obj/item/weapon/holo/esword/attack(target as mob, mob/user as mob)
```

Comment:
Really if you're gonna fix this in such a half assed way do it with the other holodeck items too.